GS03FLMA/GS03FLMA_R00003_ingest.csv

```diff
@@ -1,34 +1,34 @@
+#,/omc_data/whoi/OMC/GS03FLMA/R00003/cg_data/MSIOC_sn/CTD/*.DAT,GS03FLMA-RIM01-02-CTDMOG041,recovered_host,Available,Data assigned to refdes after ingestion with wildcard refdes
+mi.dataset.driver.ctdmo_ghqr.sio.ctdmo_ghqr_ct_recovered_driver,/omc_data/whoi/OMC/GS03FLMA/R00003/instruments/CTDMO_sn37-12330/*.hex,GS03FLMA-RIM01-02-CTDMOG041,recovered_inst,Available,
+#,/omc_data/whoi/OMC/GS03FLMA/R00003/cg_data/MSIOC_sn/CTD/*.DAT,GS03FLMA-RIM01-02-CTDMOG042,recovered_host,Available,Data assigned to refdes after ingestion with wildcard refdes
+mi.dataset.driver.ctdmo_ghqr.sio.ctdmo_ghqr_ct_recovered_driver,/omc_data/whoi/OMC/GS03FLMA/R00003/instruments/CTDMO_sn37-12370/*.hex,GS03FLMA-RIM01-02-CTDMOG042,recovered_inst,Available,
+#,/omc_data/whoi/OMC/GS03FLMA/R00003/cg_data/MSIOC_sn/CTD/*.DAT,GS03FLMA-RIM01-02-CTDMOG043,recovered_host,Available,Data assigned to refdes after ingestion with wildcard refdes
+mi.dataset.driver.ctdmo_ghqr.sio.ctdmo_ghqr_ct_recovered_driver,/omc_data/whoi/OMC/GS03FLMA/R00003/instruments/CTDMO_sn37-12332/*.hex,GS03FLMA-RIM01-02-CTDMOG043,recovered_inst,Available,
+#,/omc_data/whoi/OMC/GS03FLMA/R00003/cg_data/MSIOC_sn/CTD/*.DAT,GS03FLMA-RIM01-02-CTDMOG044,recovered_host,Available,Data assigned to refdes after ingestion with wildcard refdes
+mi.dataset.driver.ctdmo_ghqr.sio.ctdmo_ghqr_ct_recovered_driver,/omc_data/whoi/OMC/GS03FLMA/R00003/instruments/CTDMO_sn37-12372/*.hex,GS03FLMA-RIM01-02-CTDMOG044,recovered_inst,Available,
+#,/omc_data/whoi/OMC/GS03FLMA/R00003/cg_data/MSIOC_sn/CTD/*.DAT,GS03FLMA-RIM01-02-CTDMOG045,recovered_host,Available,Data assigned to refdes after ingestion with wildcard refdes
+mi.dataset.driver.ctdmo_ghqr.sio.ctdmo_ghqr_ct_recovered_driver,/omc_data/whoi/OMC/GS03FLMA/R00003/instruments/CTDMO_sn37-12336/*.hex,GS03FLMA-RIM01-02-CTDMOG045,recovered_inst,Available,
+#,/omc_data/whoi/OMC/GS03FLMA/R00003/cg_data/MSIOC_sn/CTD/*.DAT,GS03FLMA-RIM01-02-CTDMOG046,recovered_host,Available,Data assigned to refdes after ingestion with wildcard refdes
+mi.dataset.driver.ctdmo_ghqr.sio.ctdmo_ghqr_ct_recovered_driver,/omc_data/whoi/OMC/GS03FLMA/R00003/instruments/CTDMO_sn37-12374/*.hex,GS03FLMA-RIM01-02-CTDMOG046,recovered_inst,Available,
+#,/omc_data/whoi/OMC/GS03FLMA/R00003/cg_data/MSIOC_sn/CTD/*.DAT,GS03FLMA-RIM01-02-CTDMOG047,recovered_host,Available,Data assigned to refdes after ingestion with wildcard refdes
+mi.dataset.driver.ctdmo_ghqr.sio.ctdmo_ghqr_ct_recovered_driver,/omc_data/whoi/OMC/GS03FLMA/R00003/instruments/CTDMO_sn37-12314/*.hex,GS03FLMA-RIM01-02-CTDMOG047,recovered_inst,Available,
+#,/omc_data/whoi/OMC/GS03FLMA/R00003/cg_data/MSIOC_sn/CTD/*.DAT,GS03FLMA-RIM01-02-CTDMOG048,recovered_host,Available,Data assigned to refdes after ingestion with wildcard refdes
+#mi.dataset.driver.ctdmo_ghqr.sio.ctdmo_ghqr_ct_recovered_driver,/omc_data/whoi/OMC/GS03FLMA/R00003/instruments/CTDMO_sn37-12331/*.hex,GS03FLMA-RIM01-02-CTDMOG048,recovered_inst,Available,

```

Comment:
I think this line should be uncommented.

Reply:
Agreed